src/main/java/com/educare/unitylend/dao/BorrowRequestRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.educare.unitylend.dao;
+
+import com.educare.unitylend.model.BorrowRequest;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.annotations.Select;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Mapper
+@Repository
+public interface BorrowRequestRepository {
+
```

Comment:
1. Add method to fetch borrow request based on community_id, This is required for lenders page.
2. Modify the schema to have a borrow request multiple communities, You can have a separate table which can have the mapping of community to Borrow request.